packages/govuk-frontend/src/govuk/tools/rebrand.unit.test.js

```diff
@@ -0,0 +1,144 @@
+const { compileSassString } = require('@govuk-frontend/helpers/tests')
+const { outdent } = require('outdent')
+
+describe('@mixin _govuk-rebrand', () => {
+  it('wraps arbitrary properties in a class', async () => {
+    const sass = `
+      $govuk-suppressed-warnings: ("legacy-organisation-colours");
+      @import "base";
+
+      .foo {
+        border-width: 1px;
+        border-colour: #fff;
+
+        @include _govuk-rebrand() {
+          border-width: 10px;
+          border-colour: #000;
+        }
+      }
+    `
+
+    await expect(compileSassString(sass)).resolves.toMatchObject({
+      css: outdent`
+        .foo {
+          border-width: 1px;
+          border-colour: #fff;
+        }
+        .govuk-template--rebranded .foo {
+          border-width: 10px;
+          border-colour: #000;
+        }
+      `
+    })
+  })
+
+  describe('`$property` argument', () => {
+    it('renders  both for it original and rebranded version', async () => {
```

Comment:
```suggestion
    it('renders both original and rebranded version', async () => {
```

I'm guessing this phrase got a little mangled in an edit? I've taken a guess at what it should say.

Reply:
Oh yeah, good spot! I'll fix that, cheers!